### PR TITLE
Tinybird charts

### PR DESCRIPTION
**Pull request by rmorehig**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have update the [CHANGELOG](https://github.com/tinybirdco/web-analytics-starter-kit/blob/main/CHANGELOG.md)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alejandromav** [comment]
Couple of differences I see:

1- I cannot click on these links anymore:
image


2- The country names and flags here:
image

File: dashboard/components/Widgets.tsx [hunk 1]
```diff
@@ -9,58 +24,216 @@ const enum WidgetHeight {
...
+  const { query } = useRouter()
+  const { host, token } = getConfig(
+    typeof query === 'string' ? query : undefined
+  )
+  function buildEndointUrl(host: string, endpoint: string) {
+    const apiUrl =
+      {
+        'https://ui.tinybird.co': 'https://api.tinybird.co',
+        'https://ui.us-east.tinybird.co': 'https://api.us-east.tinybird.co',
+      }[host] ?? host
```
Comment (pjosh) [inline dashboard/components/Widgets.tsx:37]: I saw that the `host` is defined by an env var or the url host. Is this something that needs to be done to maintain some sort of backwards compatibility?

File: dashboard/lib/api.ts [hunk 2]
```diff
@@ -43,6 +43,30 @@ export async function client<T>(
...
+      Authorization: `Bearer ${token}`,
+    },
+    ...params,
+  })
+  const data = (await response.json()) as ClientResponse<T>
+
+  if (!response.ok) {
+    throw new QueryError(data?.error ?? 'Something went wrong', response.status)
+  }
+  console.log(data)
```
Comment (pjosh) [inline dashboard/lib/api.ts:69]: console.log

File: dashboard/components/Widgets.tsx (hunk 1 above)
Comment (rmorehig) [reply dashboard/components/Widgets.tsx:37]: This is the old way of defining the api host. We can fix this in another PR.

**Pull request merged**